lib/cli/interactive-setup/service.js

```diff
@@ -130,14 +145,34 @@ module.exports = {
         projectDir,
         projectName,
       });
-    } else {
-      const projectType = await projectTypeChoice();
-      if (projectType === 'other') {
+    } else if (context.options['template-url']) {
+      projectName = await resolveProjectNameInput(context.options, workingDir);
+      projectDir = join(workingDir, projectName);
+      const templateUrl = context.options['template-url'];
+      try {
         process.stdout.write(
-          '\nRun “serverless create --help” to view available templates and create a new project ' +
-            'from one of those templates.\n'
+          `\n${chalk.green(`Downloading template from provided url: ${templateUrl}...`)}\n`
```

Comment:
We use green to communicate that some operation was successful.

In that case we should probably stick to white (?)